apiserver/params/migration.go

```diff
@@ -94,10 +94,11 @@ type SerializedModelTools struct {
 // SerializedModelResource holds the details for a single resource for
 // an application in a serialized model.
 type SerializedModelResource struct {
-	Application         string                          `json:"application"`
-	Name                string                          `json:"name"`
-	ApplicationRevision SerializedModelResourceRevision `json:"application-revision"`
-	CharmStoreRevision  SerializedModelResourceRevision `json:"charmstore-revision"`
+	Application         string                                     `json:"application"`
+	Name                string                                     `json:"name"`
+	ApplicationRevision SerializedModelResourceRevision            `json:"application-revision"`
+	CharmStoreRevision  SerializedModelResourceRevision            `json:"charmstore-revision"`
+	UnitRevisions       map[string]SerializedModelResourceRevision `json:"unit-revisions"`
```

Comment:
Do you mean the charmstore revisions or the unit revisions. The unit revisions are definitely being used.

The charmstore revisions are returned but aren't used at this stage. It would be good to set them for completeness but I haven't had time to do it. I'd like to leave then to make it easy to add support for this later.